### ISSUE TITLE
BUG: in asv continuous on latest master

### ISSUE THREAD

**Issue by jreback** [spacetelescope/asv#317]
```
· Creating environments
· Discovering benchmarks
·· Uninstalling from py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt.
·· Building for py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt................................
·· Installing into py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt..
· Running 10 total benchmarks (2 commits * 1 environments * 5 benchmarks)
[  0.00%] · For pandas commit hash ab488b89:
[  0.00%] ·· Building for py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt................................
[  0.00%] ·· Benchmarking py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt
[ 10.00%] ··· Running binary_ops.timestamp_tz_ops_diff1.time_timestamp_ops_diff1                                                                                                                                            179.33?s
[ 20.00%] ··· Running binary_ops.timestamp_tz_ops_diff2.time_timestamp_ops_diff2                                                                                                                                            247.17ms
[ 30.00%] ··· Running binary_ops.timestamp_tz_series_compare.time_timestamp_series_compare                                                                                                                                  208.09?s
[ 40.00%] ··· Running timeseries.dti_reset_index_tz.time_dti_reset_index_tz                                                                                                                                                 451.28?s
[ 50.00%] ··· Running timeseries.timeseries_timestamp_tzinfo_cons.time_timeseries_timestamp_tzinfo_cons                                                                                                                       8.37?s
[ 50.00%] · For pandas commit hash f82e1778:
[ 50.00%] ·· Building for py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt................................
[ 50.00%] ·· Benchmarking py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt
[ 60.00%] ··· Running binary_ops.timestamp_tz_ops_diff1.time_timestamp_ops_diff1                                                                                                                                            159.14ms
[ 70.00%] ··· Running binary_ops.timestamp_tz_ops_diff2.time_timestamp_ops_diff2                                                                                                                                            157.42ms
[ 80.00%] ··· Running binary_ops.timestamp_tz_series_compare.time_timestamp_series_compare                                                                                                                                    1.59ms
[ 90.00%] ··· Running timeseries.dti_reset_index_tz.time_dti_reset_index_tz                                                                                                                                                   6.04ms
[100.00%] ··· Running timeseries.timeseries_timestamp_tzinfo_cons.time_timeseries_timestamp_tzinfo_cons                                                                                                                       8.46?s
BENCHMARK                                    BEFORE      AFTER     FACTOR
Traceback (most recent call last):
  File "/Users/jreback/miniconda/bin/asv", line 9, in <module>
    load_entry_point('asv==0.2.dev794+8819998b', 'console_scripts', 'asv')()
  File "/Users/jreback/miniconda/lib/python2.7/site-packages/asv-0.2.dev794+8819998b-py2.7.egg/asv/main.py", line 36, in main
    result = args.func(args)
  File "/Users/jreback/miniconda/lib/python2.7/site-packages/asv-0.2.dev794+8819998b-py2.7.egg/asv/commands/__init__.py", line 48, in run_from_args
    return cls.run_from_conf_args(conf, args)
  File "/Users/jreback/miniconda/lib/python2.7/site-packages/asv-0.2.dev794+8819998b-py2.7.egg/asv/commands/continuous.py", line 50, in run_from_conf_args
    show_stderr=args.show_stderr, bench=args.bench, machine=args.machine
  File "/Users/jreback/miniconda/lib/python2.7/site-packages/asv-0.2.dev794+8819998b-py2.7.egg/asv/commands/continuous.py", line 141, in run
    before_display, after_display, change))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u03bc' in position 60: ordinal not in range(128)
```

**Comment by mdboom** [comment]
It looks like your console isn't set up to display Unicode (you're on a Mac, I guess).  On Macs, I routinely set the environment variable `PYTHONIOENCODING=utf-8` so this kind of stuff works.  I guess Python doesn't do that by default for historical reasons, but in 2015 it's a really safe bet and it's never given me problems.

In any event, the `print` functions in `continuous` should probably be replaced with `util.color_print` to take advantage of the encoding fallbacks there.

**Comment by jreback** [comment]
I set ``PYTHONIOENCODING=utf-8``

so works now, though I don't think these characters are interpreted correctly (should be dots?)
note that I am using and xterm on macosx (so maybe a bit odd :)


```
[jreback-~/pandas/asv_bench] asv continuous --bench tz tz
Â· Creating environments
Â· Discovering benchmarks
Â·Â· Uninstalling from py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt.
Â·Â· Building for py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt.............................
Â·Â· Installing into py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt..
Â· Running 10 total benchmarks (2 commits * 1 environments * 5 benchmarks)
[  0.00%] Â· For pandas commit hash a52d8404:
[  0.00%] Â·Â· Building for py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt.................................
[  0.00%] Â·Â· Benchmarking py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt
[ 10.00%] Â·Â·Â· Running binary_ops.timestamp_tz_ops_diff1.time_timestamp_ops_diff1                                                                                                                                            173.10Î¼s
[ 20.00%] Â·Â·Â· Running binary_ops.timestamp_tz_ops_diff2.time_timestamp_ops_diff2                                                                                                                                            238.30ms
[ 30.00%] Â·Â·Â· Running binary_ops.timestamp_tz_series_compare.time_timestamp_series_compare                                                                                                                                  194.56Î¼s
[ 40.00%] Â·Â·Â· Running timeseries.dti_reset_index_tz.time_dti_reset_index_tz                                                                                                                                                 443.71Î¼s
[ 50.00%] Â·Â·Â· Running timeseries.timeseries_timestamp_tzinfo_cons.time_timeseries_timestamp_tzinfo_cons                                                                                                                       8.09Î¼s
[ 50.00%] Â· For pandas commit hash 2d0d58c0:
[ 50.00%] Â·Â· Building for py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt................................
[ 50.00%] Â·Â· Benchmarking py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt
[ 60.00%] Â·Â·Â· Running binary_ops.timestamp_tz_ops_diff1.time_timestamp_ops_diff1                                                                                                                                                170.08ms
[ 70.00%] Â·Â·Â· Running binary_ops.timestamp_tz_ops_diff2.time_timestamp_ops_diff2                                                                                                                                                169.49ms
[ 80.00%] Â·Â·Â· Running binary_ops.timestamp_tz_series_compare.time_timestamp_series_compare                                                                                                                                        1.74ms
[ 90.00%] Â·Â·Â· Running timeseries.dti_reset_index_tz.time_dti_reset_index_tz                                                                                                                                                       5.44ms
[100.00%] Â·Â·Â· Running timeseries.timeseries_timestamp_tzinfo_cons.time_timeseries_timestamp_tzinfo_cons                                                                                                                          10.08Î¼s
BENCHMARK                                    BEFORE      AFTER     FACTOR
...compare.time_timestamp_series_compare     1.74ms   194.56Î¼s   0.11169413x
...eset_index_tz.time_dti_reset_index_tz     5.44ms   443.71Î¼s   0.08157783x
...tz_ops_diff1.time_timestamp_ops_diff1   170.08ms   173.10Î¼s   0.00101776x

SOME BENCHMARKS HAVE CHANGED SIGNIFICANTLY.
```

**Comment by mdboom** [comment]
Ok.  It looks like your xterm config doesn't support utf-8 after all.  I think #318 (or something like it) is probably the best option (or we can just remove unicode from our output altogether).

**Comment by jreback** [comment]
hmm, doesn't work for me....no big deal though.

```
Finished processing dependencies for asv==0.2.dev795+c27e60f7

[jreback-~/pandas/asv_bench] asv continuous --bench tz tz
Â· Creating environments
Â· Discovering benchmarks
Â·Â· Uninstalling from py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt.
Â·Â· Building for py2.7-Cython-matplotlib-numexpr-numpy-openpyxl-pytables-scipy-sqlalchemy-xlrd-xlwt.......^CTraceback (most recent call last):
```

**Comment by jorisvandenbossche** [comment]
Just had the same issue on windows:

```
[ 98.39%] ··· Running groupby.groupby_ngroups_10000.time_unique        860.79ms
[ 99.19%] ··· Running groupby.groupby_ngroups_10000.time_value_counts     6.13s
[100.00%] ··· Running groupby.groupby_ngroups_10000.time_var             3.12ms
BENCHMARK                                    BEFORE      AFTER     FACTOR
Traceback (most recent call last):
  File "C:\Anaconda\envs\devel\Scripts\asv-script.py", line 9, in <module>
    load_entry_point('asv==0.2.dev760-55f78f1d', 'console_scripts', 'asv')()
  File "C:\Anaconda\envs\devel\lib\site-packages\asv-0.2.dev760_55f78f1d-py2.7.e
gg\asv\main.py", line 36, in main
    result = args.func(args)
  File "C:\Anaconda\envs\devel\lib\site-packages\asv-0.2.dev760_55f78f1d-py2.7.e
gg\asv\commands\__init__.py", line 47, in run_from_args
    return cls.run_from_conf_args(conf, args)
  File "C:\Anaconda\envs\devel\lib\site-packages\asv-0.2.dev760_55f78f1d-py2.7.e
gg\asv\commands\continuous.py", line 50, in run_from_conf_args
    show_stderr=args.show_stderr, bench=args.bench, machine=args.machine
  File "C:\Anaconda\envs\devel\lib\site-packages\asv-0.2.dev760_55f78f1d-py2.7.e
gg\asv\commands\continuous.py", line 141, in run
    before_display, after_display, change))
  File "C:\Anaconda\envs\devel\lib\encodings\cp850.py", line 12, in encode
    return codecs.charmap_encode(input,errors,encoding_map)
UnicodeEncodeError: 'charmap' codec can't encode character u'\u03bc' in position
 60: character maps to <undefined>
```

Will try out the patch

**Comment by pv** [comment]
See if latest master works for you

**Comment by jorisvandenbossche** [comment]
Yes, doing that now first. Should that in principle already fix it? Or should I also need #318?

**Comment by pv** [comment]
It will likely already fix it as is, but #318 makes things more
consistent by using the unicode fallback everywhere.

**Comment by jorisvandenbossche** [comment]
yes, solved for me with master
